enterprise/maven.j2ee/src/org/netbeans/modules/maven/j2ee/ui/wizard/archetype/J2eeArchetypeFactory.java

```diff
@@ -90,8 +90,8 @@ public Archetype getAnyArchetypeFor(J2eeModule.Type projectType) {
     private static class AppClientArchetypes extends BaseJ2eeArchetypeProvider {
         @Override
         protected void setUpProjectArchetypes() {
-            addJavaEE8Archetype(Profile.JAVA_EE_8_FULL,"io.github.juneau001", "1.0.0", "webapp-javaee8"); //NOI18N
-            addMojoArchetype(Profile.JAVA_EE_7_FULL, "1.1", "appclient-javaee7"); //NOI18N
+            addJavaEE8Archetype(Profile.JAVA_EE_8_FULL,"io.github.juneau001", "1.2", "webapp-javaee8"); //NOI18N
+            addMojoArchetype(Profile.JAVA_EE_7_FULL, "1.2", "appclient-javaee7"); //NOI18N
```

Comment:
Is appclient-javaee7 archetype version upgrade required? The latest available version is 1.1.

Reply:
Thanks for checking.  I have done a basic check on some of the other archetype versions and some of them look incorrect.  For example ejb-javaee7 is coded as version 1.2, and I can only find 1.1 on mvn central.  I will look through and make necessary changes, where needed.

Reply:
Checked archetype versions against https://repo1.maven.org/maven2/org/codehaus/mojo/archetypes/ and updated accordingly.